corehq/tests/noseplugins/dividedwerun.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import types
+from hashlib import md5
+from unittest import SkipTest
+
+from nose.case import Test, FunctionTestCase
+from nose.plugins import Plugin
+from nose.suite import ContextSuite
+
+log = logging.getLogger(__name__)
+
+
+class DividedWeRunPlugin(Plugin):
```

Comment:
Is the purpose of this to allow us to split the travis tests into multiple runs?